<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaopcionesxmpp</class>
 <widget class="QMainWindow" name="ventanaopcionesxmpp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Opciones Xmpp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Metodos de transferencia de archivos</string>
      </property>
      <widget class="QRadioButton" name="checkEnBanda">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>97</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>En banda</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="checkFueraDeBanda">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Fuera de banda</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Proxy de archivos:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textoProxy">
        <property name="text">
         <string>proxy.eu.jabber.org</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="botonEstablecer">
      <property name="text">
       <string>Establecer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
